datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaTimeStamps.tsx

```diff
@@ -0,0 +1,64 @@
+import { ClockCircleOutlined } from '@ant-design/icons';
```

Comment:
Thanks for refactoring this into its own file!

Reply:
+1!